recipes/funcannot/meta.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: funcannot
+  version: "v2.8"
+
+source:
+  url: https://github.com/BioTools-Tek/funcannot/archive/master.zip
+  md5: 2e70b503a1632998deba759b50c0c07f
+   
+
+build:
+  number: 0
+  #skip: True # [osx]
+
+requirements:
+  build:
+    - gcc
```

Comment:
llvm on macos is not working?

Reply:
untested, but if it can build under gcc, why not use gcc?

Reply:
(I will specify llvm for osx, in future commits - I just want to know why llvm is preferred over a more cross-platform soln)